docs_src/microservices/device/supported/device-usb-camera/supplementary-info/advanced-options.md

```diff
@@ -1,5 +1,34 @@
 # Advanced Options
 
+## RTSP Auth
+The device service allows for rtsp stream authentication using the [rtsp-simple-server](https://github.com/aler9/mediamtx). Authentication is enabled by default.
+
+To configure the username and password for the authentication when buiding your own images, edit the fields in the 'configuration.yaml'
+!!! note 
+    Leaving the fields blank will **NOT** disable authentication. The stream will not be able to be authenticated until credentials are provided.
+
+!!! example - "Snippet from configuration.yaml"
+    ```yaml
+    ...
+    Writable:
+        LogLevel: "INFO"
+        InsecureSecrets:
+            rtspauth:
+            SecretName: rtspauth
+            SecretData:
+                username: ""
+                password: ""
+    ```
+
+To disable authentication, comment out the externalAuthenticationURL line in the `Dockerfile`.
```

Comment:
I think we should point to the location of the `Dockerfile`.

Reply:
I meant should we point to `https://github.com/edgexfoundry/device-usb-camera/blob/main/Dockerfile`?